src/control/cmd/dmg/pool.go

```diff
@@ -86,9 +72,17 @@ type PoolCreateCmd struct {
 func (c *PoolCreateCmd) Execute(args []string) error {
 	msg := "SUCCEEDED: "
 
-	scmBytes, nvmeBytes, err := calcStorage(c.log, c.ScmSize, c.NVMeSize)
+	scmBytes, err := humanize.ParseBytes(c.ScmSize)
```

Comment:
I think you're right about this. That is a point I was trying to raise with my comments in DAOS-4049. From a usability standpoint, I don't think it's useful to distinguish between DCPM and NVMe in terms of how the storage is displayed or specified. Humans prefer to work with base-10, in my experience. What I really want to avoid is converting base-10 inputs to base-2 values behind the scenes, because we'll be back to the same place we were before. In my opinion, we should display usable storage (i.e. the storage available to create pools) entirely in base-10, because that is what most users will use to specify the pool sizes.

Reply:
@kjacque yes the behaviour has been corrected, will now correctly interpret either IEC or SI units into bytes, whichever is specified. @mjmac memory capacity is specified by manufacturers in IEC units and storage capacity is specified in SI, we should mirror those conventions when displaying device capacity. this is what was originally requested by @johannlombardi  . with regard to these specific comments I don't think any changes are actually required, please correct me if I'm wrong.

Reply:
I agree that displaying IEC units makes sense in the context of displaying individual memory modules, before SCM prep has been done. After the namespaces have been created and associated with block devices, the SCM is being used as storage, not memory, and it doesn't make sense to display it with IEC units anymore. I was trying to engage in this design discussion in the ticket, before new code had been written, but I guess we'll have it here instead.

Let's think about how the output will be used. Currently we don't have a high-level `df` equivalent (perhaps something like `dmg storage query`) where we show Used/Available, but we will eventually and this discussion applies there as well. So imagine we implement this command, and the output looks something like this:

```bash
$ dmg storage query
 DAOS         SCM                NVMe
Servers  Used / Available   Used / Available
-------  ----------------   ----------------
  16     65 TiB / 47 TiB    1.38 PB / 673 TB
```

Now, as a DAOS admin, I have a request from a user to create a pool for their project. The user doesn't know anything about SCM vs NVMe, so they're just going to ask for a chunk of storage. Let's say 10TB. As an admin, I do know about SCM vs NVMe, and so I want to make sure that the project gets the correct ratio of SCM to NVMe in order to guarantee the correct performance. So I decide that the project gets a total of 10TB, but I'm going to make sure that the pool storage is backed by 25% SCM. 10TB * .25 = 2.5TB, so I want to create a pool with `--scm-size 2.5TB --nvme-size 7.5TB`.

So far so good. The math was easy, and dmg parsed the units into byte counts. The discrepancy between displayed units and input units didn't really matter, but it might be kind of weird to see that the pool's SCM portion is in TiB and have to convert between units to verify the ratio, etc. Now here's where it gets a bit more awkward. Imagine this same system is brand new and the admin wants to create a giant pool using all storage. Here's what the imaginary query command shows:

```bash
$ dmg storage query
 DAOS         SCM                NVMe
Servers  Used / Available   Used / Available
-------  ----------------   ----------------
  16      0 B / 112 TiB       0 B / 2 PB
```

So as the admin, I try this: `dmg pool create --scm-size 112TB --nvme-size 2PB` and it succeeds. But when I display the storage again, here's what it looks like:

```bash
$ dmg storage query
 DAOS           SCM                NVMe
Servers    Used / Available    Used / Available
-------   ------------------   ----------------
  16      102 TiB / 10.1 TiB      2 PB / 0 B
```

What gives? Oh, I actually have to specify tebibytes? Er, OK. I guess I'll delete the pool and try again. In the grand scheme of things, perhaps this is a minor annoyance, but it is a usability issue IMO. I don't want to belabor the point, but I could come up with other scenarios where the mismatch in units causes other problems which aren't so quickly detected.

None of this is broken, exactly, but it's not as usable as it could be, IMO. Is there some reason I'm not seeing where it's actually preferable/useful to have the SCM storage displayed in IEC units?

Reply:
Hmm. And here's another usability issue... I keep forgetting that the parameters specified to `dmg pool create` are storage allocations _per server_, not per pool. But that's a different issue. It does argue for keeping the display/math as simple as possible.

Reply:
Created https://jira.hpdd.intel.com/browse/DAOS-4170 to explore further dmg improvements.

Reply:
amended to only show IEC units for SCM module capacity